### ISSUE TITLE
It connects but...

### ISSUE THREAD

**Issue by thedudemeister08** [KozmikNano/KozmikServices#4]
It connects but it doesn't unblock websites. I have tried doing chrome://restart but that doesn't work either.

**Comment by KozmikNano** [comment]
Are you using the DNS version? You need that one for the unblocking to work

**Comment by thedudemeister08** [comment]
> Are you using the DNS version? You need that one for the unblocking to work

I've tried using both. They both connect fine but nothing is unblocked

**Comment by KozmikNano** [comment]
> I've tried using both. They both connect fine but nothing is unblocked

Whats your blocker?

**Comment by thedudemeister08** [comment]
> > I've tried using both. They both connect fine but nothing is unblocked
> 
> 
> 
> Whats your blocker?

GoGuardian and iBoss

**Comment by KozmikNano** [comment]
Ibossy may be your issue, is iboss setup as an extension?

**Comment by thedudemeister08** [comment]
> Ibossy may be your issue, is iboss setup as an extension?

Yeah and an app plugin but the other one was disabling it so idk why this one isn't

**Comment by KozmikNano** [comment]
> Yeah and an app plugin but the other one was disabling it so idk why this one isn't

Sorry for asking but what do you mean by the other one?

**Comment by thedudemeister08** [comment]
> > Yeah and an app plugin but the other one was disabling it so idk why this one isn't
> 
> 
> 
> Sorry for asking but what do you mean by the other one?

The other VPN

**Comment by Sreekar617** [comment]
I have the same issue, I have Securly as an extension blocking websites, even if I have the DNS version active.
